### PR TITLE
Milestone 2: Portfolio Setup on Mobile version

### PR DESCRIPTION
**Pull request by smas0900**
In this milestone we:

- Added Html &CSS file.
- Added Logos..
- Merge the project from previous Project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+            </div>
+            
+            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list">
+                <li class="box"><button type="button">html</button></li>
+                <li class="box"><button type="button">css</button></li>
+                <li class="box"><button type="button">javaScript</button></li>
+            </ul>
+            
+            <button class ="card_see" type="button">See Project</button>
```
Comment (AnselemOdims) [inline index.html:51]: - [ ] Good job using a `button` tag for this but please kindly ensure that the style for its different states(hover, pressed and inactive) are taken care of.

image

File: index.html [hunk 2]
```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+                <p>CANOPY</p>
+                <span class="dot"></span>
+                <p class="sec_color1">Back End Dev</p>
+                <span class="dot"></span>
+                <p class="sec_color2">2015</p>
+            </div>
+            
+            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list">
+                <li class="box"><button type="button">html</button></li>
```
Comment (AnselemOdims) [inline index.html:46]: - [ ] Please kindly make sure that the same font used in the original design is also used.
- [ ] Please ensure that the cards have a `border-radius` as it is in the original design

### Your Design
image

### Original Design
image

File: index.html [hunk 3]
```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+        <p class="para">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="">LET'S CONNECT</a>
+        <ul class="icons">
+            <li class="icons_list"><img src="./images/Vector-4.png" alt="navigation icon" ></li>
+            <li class="icons_list"><img src="./images/Vector-2.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector-1.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector-3.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector.png" alt="navigation icon"></li>
+        </ul>
+       <button class="btn" type="button">Get my resume</button>
```
Comment (AnselemOdims) [inline index.html:130]: - [ ] Please kindly make sure the same font used in the original design is used here also
- [ ] Please ensure that there is enough padded space at the top, just before the `About Myself` heading
- [ ] Please ensure that the button style reflects the different states(hover, pressed and inactive)

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @smas0900 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @ ,

Congratulations on completing this project! 👏🏼 🎉

- [ ] _[OPTIONAL]_ Please consider using [responsively](https://responsively.app/) when developing your app, some styling edge cases are not correctly maintained on very small screen sizes ~320 px.
You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — COMMENTED** [review]
Hi @smas0900 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, whose PR has already been merged a long ago

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by smas0900** [comment]
I did this project before now I am repeating the module
So what should i do? 
@BertrandConxy

**Comment by BertrandConxy** [comment]
oh.. I think when you have repeated the module,  the solo projects which have been approved already,  are supposed to be approved again.
Please request another review and inform the user about your case

**Review by MaganaAsiati — COMMENTED** [review]
# Hi @smas0900 
## Invalid Code Review Request :negative_squared_cross_mark:
made a great effort in this project, unfortunately, I cannot proceed to review your code.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏





- [ ] You have submitted a project  which has been already approved, kindly make sure you submit the right  link and make your pull request more descriptive 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
Cheers and Happy coding!👏👏👏
--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by smas0900** [comment]
@MaganaAsiati can you explain what should i do now did u read @BertrandConxy comments?

**Pull request merged**